config/betamocks/services_config.yml

```diff
@@ -22,6 +34,10 @@
       :path: <%= "/#{Settings.ask_va_api.crm_api.tenant_id}/oauth2/v2.0/token" %>
       :file_path: "/ask_va/token/default"
       :response_delay: 0.3
+    - :method: :post
+      :path: <%= "/#{Settings.travel_pay.veis.tenant_id}/oauth2/token" %>
+      :file_path: '/ask_va/token/default'
```

Comment:
Why is the file path `ask_va`?

Reply:
I think that was just an copy-paste oversight as I was debugging [this](https://github.com/department-of-veterans-affairs/vets-api/pull/16231#discussion_r1557809134). I'll add one in our directory in the betamocks repo, even if it's just a dupicate of the other mocked token response.

Reply:
fixed in [9ea18f3](https://github.com/department-of-veterans-affairs/vets-api/pull/16231/commits/9ea18f3703f334db513c651eb2b69d18b011f538)